### DISCUSSION TITLE
Should we consider transferring the project to https://github.com/django-commons ?

### DISCUSSION THREAD

**Discussion by ticosax** [gizmag/django-fsm-log#204] [General]
For the reason the project is staled https://github.com/jazzband/django-fsm-log/pull/192
Bad news: we are still depending on the roadies.
Good news: I still have admin rights on pypi.
Worse case scenario we do an hard fork and advertise on the readme the new location of the project.

**Comment by tim-schilling** [comment]
If you have any questions about how Django Commons works, please feel free to let me know. It's not perfect by any means. I help maintain the debug toolbar which moved over a few months ago, so any problems with the infrastructure, I'm aware and wanting to fix.
